src/main/java/frc/robot/subsystems/Harvester.java

```diff
@@ -7,16 +7,15 @@
 public class Harvester extends SubsystemBase {
 
     // ESC declarations
-    private final VictorSP intakeMotor; 
+    private final VictorSP intakeMotor;
 
     // Piston declarations
-    private final Solenoid deployPistion;
-
+    private final Solenoid deployPiston;
```

Comment:
We want 2 deploy pistons: a `portDeployPiston` and a `starbordDeployPiston`

Reply:
Done by the man himself, TaiSeanDo